README.md

```diff
@@ -50,13 +50,187 @@ module "postgresql" {
 }
 ```
 
+```hcl
+
+module "opensearch_domain" {
+  source = "github.com/camunda/camunda-tf-eks-module/modules/opensearch"
+
+  domain_name     = "my-opensearch-domain"
+  subnet_ids      = module.eks_cluster.private_subnet_ids
+  security_group_ids = module.eks_cluster.security_group_ids
+  vpc_id          = module.eks_cluster.vpc_id
+  cidr_blocks      = concat(module.eks_cluster.private_vpc_cidr_blocks, module.eks_cluster.public_vpc_cidr_blocks)
+
+  instance_type   = "t3.small.search"
+  instance_count  = 4
+  ebs_volume_size = 100
+
+  advanced_security_enabled = true
+  advanced_security_internal_user_database_enabled = true
+  advanced_security_master_user_name = "admin"
+  advanced_security_master_user_password = "password"
+
+  depends_on = [module.eks_cluster]
+}
+```
+
 #### GitHub Actions
 
 You can automate the deployment and deletion of the EKS cluster and Aurora database using GitHub Actions. Below are examples of GitHub Actions workflows for deploying and deleting these resources.
```

Comment:
\+ OpenSearch now

Reply:
I'd also add that this is for testing purposes.
People should probably use something more persistent than GitHub Actions to deploy their stuff.